plugins/inputs/redfish/testdata/hp_hostname.json

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
Rename `hp_computer_system.json`

Reply:
renamed it as hp_systems.json as it sounds more appropriate.